jellyfin-core/src/main/kotlin/org/jellyfin/apiclient/discovery/RecommendedServerInfo.kt

```diff
@@ -0,0 +1,16 @@
+package org.jellyfin.apiclient.discovery
+
+import org.jellyfin.apiclient.model.api.PublicSystemInfo
+
+data class RecommendedServerInfo(
+	val address: String,
+	val responseTime: Long,
+	val score: RecommendedServerInfoScore,
+	val systemInfo: PublicSystemInfo?,
+	val parent: String?
+) {
+	/**
+	 * True when this server was not a part of the inputted addresses, false otherwise.
```

Comment:
Interestingly, "inputted" is a valid past participle of "input", but I think I'd prefer "entered" here regardless :sweat_smile:

Reply:
According to the dictionary "input" means "something that is put in" and "enter" means "to come or go in". Since the value is already there the correct word would be "input".